obst_sim:
  obst_num: 2
  traj_speed: 0.09